tests/clar_libgit2.h

```diff
@@ -97,6 +98,23 @@ GIT_INLINE(void) clar__assert_equal_oid(
 	clar__assert_equal_oid(__FILE__, __LINE__, \
 		"OID mismatch: " #one " != " #two, (one), (two))
 
+GIT_INLINE(void) clar__assert_equal_vector_str(
+	const char *file, int line,
+	const git_vector *a, const git_vector *b)
+{
+	clar__assert_equal(file, line, "Vector length mismatch: %" PRIuZ " != %" PRIuZ, 1, PRIuZ, git_vector_length(a), git_vector_length(b));
+
+	size_t i;
+	char *string;
```

Comment:
And by the way, there's some compilation error on Windows with MSVC here. Seems to be related to `size_t`, but I didn't really take a look into it.